qmxgraph/widget.py

```diff
@@ -454,6 +464,11 @@ def _on_drop(self, event):
                         tags=v.get('tags', {}),
                     )
 
+            if version in (3,):
+                custom_app_drop_events = parsed.pop('custom_app_drop_events', [])
+                if custom_app_drop_events:
+                    self.customAppDropEvent.emit(custom_app_drop_events, x, y)
```

Comment:
```suggestion
                    self.customAppDropEvent.emit(custom_app_drop_event, x, y)
```

Reply:
I actually wrote in plural to keep consistency with previous types of drops (vertices and decorations). What do you think?

Reply:
Actually I just read your other suggestion, I'll rename to _custom_app_drop_data_

Reply:
It is just that in the other cases they are actually plural: we can post multiple vertices, but we only have a single data in fact.

Great, reanming to `custom_app_drop_event_data` and referencing a generic *object* is the way to go. 👍